suites/pacific/ceph_volume/tier-2_ceph_volume.yaml

```diff
@@ -0,0 +1,83 @@
+#===============================================================================================
```

Comment:
Lets change the test suite description. Add below details - 

- Test Suite - Description
- Setup - Test cases which are part of cluster setup
- Test Cases - List of test cases 

Refer - https://github.com/red-hat-storage/cephci/pull/2382/files#diff-ac5017c9a9c4124edf8ca6cdbc4ddb6f437af3a0db36126e544e5be473f6fbd1